source/_integrations/unifiprotect.markdown

```diff
@@ -213,6 +213,14 @@ Your main UniFi Protect NVR device also gets a number of diagnostics sensors tha
 * **Disk Health**: Each disk installed in your NVR will have a disk health sensor. These are simple good/bad sensors and the order is not promised to match the order in UniFi OS. Disk model number is provided as a state attribute though to help map sensor to disk.
 * **Utilization and Storage Sensors**: Several other sensors are also added for uptime, hardware utilization, and distribution details of the video on disk.
 
+## Ignoring Devices
+
+If for some reason you want to not have a UniFi Protect device appear in Home Assistant, you can simply "delete" the device from Home Assistant.
+
+![Delete Device](/images/integrations/unifiprotect/delete_device.png)
+
+This will ignore the MAC address of the device so it is never added back to Home Assistant again. If you change your mind and want to add the device back to Home Assistant, there is an ignored devices Config Option with the a comma separated list of devices you have ignored. Due to an [existing frontend bug](https://github.com/home-assistant/frontend/issues/9506), you may have to set the value to a single space (`" "` without quotes) to clear out all of the existing MAC addresses.
```

Comment:
I simplified it a bit, and removed the frontend part. We don't document bugs.

```suggestion
This will ignore the device, preventing it from being added back to Home Assistant again. If you change your mind later, there is an ignored devices configuration option with a comma-separated list of devices you have ignored.
```